config/jobs/image-pushing/k8s-staging-sig-storage.yaml

```diff
@@ -321,6 +321,46 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-sig-storage-gcb
               - --env-passthrough=PULL_BASE_REF
               - .
+  kubernetes-csi/external-snapshot-metadata:
+    - name: post-external-snapshot-metadata-push-images
+      rerun_auth_config:
+        github_team_slugs:
+          - org: kubernetes
+            slug: release-managers
+          - org: kubernetes
+            slug: test-infra-admins
+          - org: kubernetes
+            slug: sig-storage-image-build-admins
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-storage-image-build, sig-k8s-infra-gcb
+      decorate: true
+      decoration_config:
+        timeout: 240m
+        grace_period: 15m
+      branches:
+        # For publishing canary images.
+        - ^master$
+        - ^release-
```

Comment:
Hey @hairyhum @xing-yang ,

The development branch for https://github.com/kubernetes-csi/external-snapshot-metadata is `main` and not master `master` as with other sidecars. 

The build job will not get triggered due to this difference ? 

https://github.com/kubernetes-csi/external-snapshot-metadata 
I don't this job being triggered for the pr that merged 3 days ago.

cc @PrasadG193 @carlbraganza